packages/core-bridge/index.d.ts

```diff
@@ -145,17 +148,48 @@ export interface TelemetryOptions {
 
   /**
    * Control where to send Rust Core logs
-   *
-   * @default log to console
    */
-  logging?: Logger;
+  logging?:
+    | {
+        /**
+         * A string in (env filter format)[https://docs.rs/tracing-subscriber/0.2.20/tracing_subscriber/struct.EnvFilter.html]
+         * which determines the verboseness of logging output.
+         *
+         * You can use {@link Runtime.makeTracingFilterString()} to easily build a correctly formatted filter
+         * string based on desired log level for Core SDK and other native packages.
+         *
+         * @default `makeTracingFilterString('INFO', 'INFO')`
+         */
+        filter?: string;
+
+        /**
+         * A string in (env filter format)[https://docs.rs/tracing-subscriber/0.2.20/tracing_subscriber/struct.EnvFilter.html]
+         * which determines the verboseness of logging output.
+         *
+         * You can use {@link Runtime.makeTracingFilterString()} to easily build a correctly formatted filter
+         * string based on desired log level for Core SDK and other native packages.
+         *
+         * @default `console`
+         */
+        mode?: 'console' | 'forward';
+      }
+    | Logger;
```

Comment:
Looks like `Logger` is deprecated but isn't marked as such.
I'm considering marking all of the telemetry options experimental now but it might be too late for that.

Reply:
Based on my comment above, I'm leaning towards preserving the structure we had before instead of redefining logging altogether.

Reply:
>Looks like Logger is deprecated but isn't marked as such.

I intentionally deprecated the _properties_ inside *Logger interfaces (`console` and `forward`), rather than the _interfaces_ themselves, because I'm worried that otherwise, the compiler might incorrectly match `logger: {}` to the deprecated `ConsoleLogger` type and report deprecation.

Reply:
>I'm considering marking all of the telemetry options experimental now but it might be too late for that.

I don't think it is too late for that, and I see the value of marking TelemetryOptions as such. Will do.